go.mod

```diff
@@ -40,12 +35,10 @@ require (
 	github.com/google/uuid v1.0.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/graph-gophers/graphql-go v0.0.0-20200309224638-dae41bde9ef9
-	github.com/graph-gophers/graphql-transport-ws v0.0.2 // indirect
 	github.com/hashicorp/vault/api v1.0.4
 	github.com/minio/minio-go/v6 v6.0.55
 	github.com/mitchellh/panicwrap v1.0.0
 	github.com/paulmach/go.geojson v0.0.0-20170327170536-40612a87147b
-	github.com/pierrec/lz4 v2.6.0+incompatible // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/pkg/profile v1.2.1
 	github.com/prometheus/client_golang v1.11.1
```

Comment:
I suspect we should upgrade this to the latest 1.14 which will include new go metrics around memory allocation and maybe GC. Minimum version in the release notes is golang 1.17 per https://github.com/prometheus/client_golang/releases so I think this will be ok.

Reply:
@damonfeldman I opened a PR for this [here](https://github.com/dgraph-io/dgraph/pull/8655).  Looks like this bumps up some of our other dependencies, in particular the grpc library.  We should be able to bring this in but we will want to make sure that there are no breakages.